db/migration/src/m20240226_000006_create_peer_store.rs

```diff
@@ -0,0 +1,90 @@
+use sea_orm_migration::prelude::*;
+use sea_orm_migration::sea_orm::prelude::ChronoDateTimeUtc;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(NetworkPeer::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(NetworkPeer::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(
+                        ColumnDef::new(NetworkPeer::PeerId)
+                            .binary_len(40)
+                            .not_null()
+                            .unique_key(),
+                    )
+                    .col(ColumnDef::new(NetworkPeer::MultiAddresses).string().not_null())
```

Comment:
I think this can be null in some cases, maybe a default `"[]"` would be better.

Reply:
Why not a json value here? That would be queryable even.

Reply:
Changed it to json, but the default will be driven by the app.